build/openresty/BUILD.openresty.bazel

```diff
@@ -197,6 +234,12 @@ CONFIGURE_OPTIONS = [
 }) + select({
     "@kong//:wasmx_flag": [
         "--add-module=$$EXT_BUILD_ROOT$$/external/ngx_wasm_module",
+        # this is a workaround for rhel/centos builds
```

Comment:
FYI: @curiositycasualty @fffonion

I don't know enough to judge the level of risk/impact of this change. If necessary I'm sure it could be limited to just centos/rhel-based builds, but I opted to start with the simple approach for now.